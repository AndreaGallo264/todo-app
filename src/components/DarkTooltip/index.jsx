import { Tooltip, makeStyles } from "@material-ui/core";

const useDarkStyle = makeStyles((theme) => ({
  arrow: {
    color: theme.palette.common.black,
  },
  tooltip: {
    backgroundColor: theme.palette.common.black,
  },
}));

export const DarkTooltip = (props) => {
  const classes = useDarkStyle();
  return <Tooltip arrow classes={classes} {...props} />;
};
