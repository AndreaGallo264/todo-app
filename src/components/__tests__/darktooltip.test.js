import React from "react";
import renderer from "react-test-renderer";
import { DarkTooltip } from "../DarkTooltip";

describe("Dark tooltip component", () => {
  test("Matches snapshot", () => {
    const tree = renderer
      .create(
        <DarkTooltip title="Add">
          <p>Prueba</p>
        </DarkTooltip>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
