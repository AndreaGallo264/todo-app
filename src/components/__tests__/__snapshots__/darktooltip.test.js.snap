// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dark tooltip component Matches snapshot 1`] = `
<p
  aria-describedby={null}
  className=""
  onBlur={[Function]}
  onFocus={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  title="Add"
>
  Prueba
</p>
`;
